maintainers/maintainer-list.nix

```
@@ -450,6 +450,12 @@
     githubId = 2335822;
     name = "Alexandre Esteves";
   };
+  alexnortung = {
+    name = "alexnortung";
+    email = "alex_nortung@live.dk ";
```

This should also be in a separate commit: `maintainers: add alexnortung`
```suggestion
    email = "alex_nortung@live.dk";
```